modules/ROOT/pages/register-server.adoc

```diff
@@ -3,20 +3,21 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-If you encounter an error when performing the steps in xref:runtime-manager::servers-create.adoc[Create a Server in Runtime Manager]:
+If you encounter a Mule runtime engine registration error when xref:runtime-manager::servers-create.adoc[creating a server in Runtime Manager], you may need to configure Mule to trust the TLS certificates you are using. 
 
-Before registering Mule runtime engine, you must configure Mule to trust the TLS certificates you configured:
+To do so:
+
+. Get the public certificate for Anypoint Platform Private Cloud Edition.
+. Add the certificate to the truststore of the JVM you are using for the Mule runtime engine.
+
```

Comment:
I also want to say that "you are using" (twice above) should be "you use," but don't feel particularly strongly about that....